src/resources/views/post/show.blade.php

```diff
@@ -0,0 +1,14 @@
+<ul class="list-group">
+    @foreach ($posts as $post)
+        <li class="list-group-item" style="background-color: #FFFFF0">
+            <h5 class="card-title">{{ $post->user->name }}</h5>
+            <p class="card-text">{{ $post->content }}</p>
+            <img src="{{ asset(''.$post->image) }}" alt="Post Image" class="img-fluid">
```

Comment:
''?

Reply:
What if the post doesnt have an image?

Reply:
Regarding on the image sir both storing and displaying po, i'm currently working on that po. May problem po kasi sa storage link that's why 'di ko pa po muna pinush through this PR.